lib/distrobaker.py

```diff
@@ -470,7 +470,7 @@ def sync_repo(comp, ns='rpms', nvr=None):
     else:
         logger.debug('Attempting to synchronize the %s/%s branches using the clean pull mechanism.', ns, comp)
         try:
-            repo.git.pull('--ff-only', bscm['ref'])
+            repo.git.pull('--ff-only', 'source', bscm['ref'])
```

Comment:
Can we have this as a variable so if we decide we need to change the remote name for some reason, it only needs to happen in one place?

Reply:
It does not.